resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -51,4 +51,11 @@ ENV SPARK_HOME /opt/spark
 
 WORKDIR /opt/spark/work-dir
 
+# add the following two lines before ENTRYPOINT
+RUN rm $SPARK_HOME/jars/kubernetes-client-*.jar
+RUN rm $SPARK_HOME/jars/guava-14.0.1.jar
+ADD https://repo1.maven.org/maven2/io/fabric8/kubernetes-client/4.4.2/kubernetes-client-4.4.2.jar $SPARK_HOME/jars
```

Comment:
4.4.2? why this version and not the one we have in dm?

Reply:
yes. I had exactly the same question. but was somehow to lazy to get it why. i copied over the lines from the image definition we used until today.